src/pages/concept-list-page/concept-list-page.ts

```diff
@@ -3,18 +3,22 @@ import { compose, withProps } from 'recompose';
 
 import { ConceptListPagePure } from './concept-list-page-pure';
 import { conceptListResolver } from './concept-list-resolver';
-import { requirePermissionDecorator } from '../../auth/require-permission-decorator';
-import { PERMISSION_READ } from '../../auth/auth-service';
+import { hasPermissionForResource, logout, PERMISSION_READ } from '../../auth/auth-service';
```

Comment:
Convert PERMISSION_READ to Permission enum with READ, WRITE and ADMIN properties

Reply:
this is temporary refactoring, better implementation in next commit